integrations/cdi/datasource-ucp/src/main/resources/META-INF/native-image/io.helidon.integrations.cdi/helidon-integrations-cdi-datasource-ucp/native-image.properties

```diff
@@ -14,7 +14,5 @@
 # limitations under the License.
 #
 
-#
-# Graal native image supports additional configuration from this property file.
-#
-Args=-H:+RemoveSaturatedTypeFlows
+Args=-H:+RemoveSaturatedTypeFlows \
```

Comment:
See https://www.graalvm.org/release-notes/20_2/#:~:text=Enabled%20%2DH%3A%2BRemoveSaturatedTypeFlows%20by%20default%2C%20which%20should%20result%20in%20faster%20native%20image%20generation%20and%20use%20less%20memory%20for%20the%20generation.; I think you can remove this

Reply:
Thanks, I have moved the file along with connection pooling module and removed the default option from config.